extensions-core/kafka-indexing-service/src/test/java/org/apache/druid/indexing/kafka/supervisor/KafkaSupervisorTest.java

```diff
@@ -585,6 +585,34 @@ public void testLatestOffset() throws Exception
     );
   }
 
+  /**
+   * Test if always use earliest offset on newly discovered partitions
```

Comment:
This comment is no longer accurate? Maybe should be something like "Test if new partitions are discovered" or even better, rename the test method so that it's self-documenting and then remove the comment.